CHANGES.md

```diff
@@ -1,3 +1,10 @@
+## Version 0.11.0
+## Version 0.11.0
```

Comment:
Oops

Reply:
sheesh, that's what i get for trying to be quick :joy: